### PR TITLE
Add MicrosoftOutlookFiles and MacOSSafariDownloads artifact

### PR DESCRIPTION
**Pull request by Karneades**

### CODE REVIEW

File: data/applications.yaml [hunk 1]
```diff
@@ -34,6 +34,19 @@ sources:
 supported_os: [Darwin, Windows]
 urls: ['https://github.com/mac4n6/macMRU-Parser']
 ---
+name: MicrosoftOutlookFiles
+doc: Microsoft Outlook Files
+sources:
+- type: FILE
+  attributes:
+    paths:
+      - '%%users.homedir%%/AppData/Local/Microsoft/Outlook/*'
```
Comment (joachimmetz) [inline data/applications.yaml]: Please make this glob more specific such as *.pst *.pab *.ost

File: data/applications.yaml [hunk 2]
```diff
@@ -34,6 +34,19 @@ sources:
 supported_os: [Darwin, Windows]
 urls: ['https://github.com/mac4n6/macMRU-Parser']
 ---
+name: MicrosoftOutlookFiles
```
Comment (joachimmetz) [inline data/applications.yaml]: Maybe make this an artifacts list and split this in
MicrosoftOutlookPABFiles
MicrosoftOutlookPSTFiles
MicrosoftOutlookOSTFiles

File: data/macos.yaml [hunk 3]
```diff
@@ -532,6 +532,20 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Recent_Items'
 ---
+name: MacOSSafariDownloads
```
Comment (joachimmetz) [inline data/macos.yaml]: Please move this definition to webbrowser.yaml and rename it to SafariDownloads

add it also to BrowserHistory

what about safari on Windows, any idea what the path of this file is?

**Comment by Karneades** [comment]
@joachimmetz Did the mentioned changes. Does it match your expectations?

**Pull request merged**